CONTENTS.md

```diff
@@ -0,0 +1,423 @@
+
+Checks for usual mistakes and problems in the source packages submitted by users. Used also as
+review bot that assigns reviews (?).
+
+* Sources: [check_source.py](check_source.py) and [check_source.pl](check_source.pl)
+* Documentation: [docs/check_source.asciidoc](docs/check_source.asciidoc)
+* Package: openSUSE-release-tools-check-source
+* Usage: [gocd](https://github.com/openSUSE/openSUSE-release-tools/search?q=path%3A%2Fgocd+osc-check_source)
+
+#### osc-origin.py
+
+Tools for working with origin information.
+
+* Sources: [osc-origin.py](osc-origin.py)
+* Documentation: [docs/origin-manager.md](docs/origin-manager.md)
+* Package: openSUSE-release-tools-origin-manager
+* Usage: [gocd](https://github.com/openSUSE/openSUSE-release-tools/search?q=path%3A%2Fgocd+osc-origin)
+
+#### osc-cycle.py
+
+Helps with OBS build cycles visualization. See the [openSUSE:Factory/standard example](https://build.opensuse.org/project/repository_state/openSUSE:Factory/standard).
+
+* Sources: [osc-cycle.py](osc-cycle.py)
+* Documentation: --
+* Package: --
+* Usage: ???
+
+#### compare_pkglist.py
+
+Compares packages status between two projects.
+
+* Sources: [compare_pkglist.py](compare_pkglist.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ???
+
+#### staging
+
+Manages staging projects.
+
+* Sources: [osc-staging.py](osc-staging.py)
+* Documentation: [docs/staging.asciidoc](docs/staging.asciidoc) and [docs/testing.asciidoc](docs/testing.asciidoc)
+* Package: osc-plugin-staging
+* Usage: ???
+
+#### status.py
+
+Checks the status of the staging workflow bots.
+
+* Sources: [status.py](status.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ???
+
+#### fcc_submitter.py
+
+Creates SR from FactoryCandidates to openSUSE Leap project for new build succeded packages.
+
+* Sources: [fcc_submitter.py](fcc_submitter.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ???
+
+#### issue-diff.py
+
+Compares packages from a project against factory for differences in referenced issues and presents
+changes to allow whitelisting before creating Bugzilla entries.
+
+
+* Sources: [issue-diff.py](issue-diff.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ???
+
+#### obs-clone.py
+
+Clones projects and dependencies between OBS instances.
+
+* Sources: [obs_clone.py](obs_clone.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ???
+
+#### obs-operator
+
+Performs staging operations as a service instead of requiring the osc staging plugin to be utilized
+directly.
+
+* Sources: [obs_operator.py](obs_operator.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ???
+
+#### scan_baselibs.py
+
+Verifies 32bit binaries were imported properly towards a project.
+
+* Sources: [scan_baselibs.py](scan_baselibs.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ???
+
+#### k8s-secret.py
+
+Applies kubernetes secrets for OSRT tool osc configuration.
```

Comment:
@coolo I kind of remember you mentioned this was a tool to maintain certain infrastructure created by @jberry-suse that is now obsolete since the introduction of gocd. Correct? If so, we can also mark this as obsolete.

Reply:
All of dist/kubernetes ran into the sysadmin wall and was never more than an experiment - and can easily be dropped IMO.